assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -78,10 +78,11 @@ spec:
 {{ if .ROKSMetricsImage }}
         - name: metrics-pusher
           image: {{ .ROKSMetricsImage }}
-{{ if .ROKSMetricsSecurity }}
+{{- if .ROKSMetricsSecurityContext }}
+{{- $probe := .ROKSMetricsSecurityContext }}
```

Comment:
variable `probe` should probably be renamed to `securityContext`.  This is a general comment.

Reply:
The `cluster-version-operator` container needs security context support.